app/views/rooms/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> All Rooms </h1>
+<div id='list_of_rooms'>
+<ul>
+<% @rooms.each do |room| %>
+<div id="room-<%= room.id %>"> 
+<li>Suite: <%= room.suite %>
+<br>
+Nightly Rate: $<%= room.rate %>
+<br>
+Hotel: <%= room.hotel.name %>
+<br>
+Total Guests: <%= room.guests.count %>
```

Comment:
This is the right functionality, but this`.count` here is technically some AR/SQL since it modifies the query. It's better practice to have this in a model method.